README.md

```diff
@@ -133,13 +128,15 @@ There are some active contribution projects underway right now:
 How Do I Interact With the Community?
 =====================================
 
-* Join the Eclipse OMR developer community [mailing list](https://accounts.eclipse.org/mailing-list/omr-dev).
-The community typically uses this list for project announcements and administrative
-discussions amongst committers.  Questions are welcome here as well.
+We are operating under the [Eclipse Code of Conduct][coc] to promote fairness, openness, and inclusion.
 
-* Join the Eclipse OMR community [Slack workspace](https://join.slack.com/t/eclipse-omr/shared_invite/enQtMzg2ODIwODc4MTAyLWFiMzZkNmNhODc5OTM0MjgwZDdjNzg5YTg5NzM0ZmEzNTIyMGViMjk1YjYwNzczYjYwODc4YTM5MDk0NjIxMjg).  You can join channels
-that interest you, ask questions, and receive answers from subject matter experts.
+[coc]: https://eclipse.org/org/documents/Community_Code_of_Conduct.php
 
+* Join the Eclipse OMR community [Slack workspace](https://join.slack.com/t/eclipse-omr/shared_invite/enQtMzg2ODIwODc4MTAyLWFiMzZkNmNhODc5OTM0MjgwZDdjNzg5YTg5NzM0ZmEzNTIyMGViMjk1YjYwNzczYjYwODc4YTM5MDk0NjIxMjg).  You can join channels that interest you, ask questions, and receive answers from subject matter experts.
+  that interest you, ask questions, and receive answers from subject matter experts.
+* Join the Eclipse OMR developer community [mailing list](https://accounts.eclipse.org/mailing-list/omr-dev).
+  The community typically uses this list for project announcements and administrative
```

Comment:
maybe want to add a "primarily" for project announcements ...  I think we want to clearly set people's expectations that the best places to engage us are via Slack and GitHub issues.